server/api/batches.py

```diff
@@ -82,9 +82,27 @@ def serialize_batch(batch: Batch) -> JSONDict:
             }
             for tally_sheet in batch.result_tally_sheets
         ],
+        "lastEditedBy": construct_batch_last_edited_by_string(batch),
```

Comment:
Decided to return a single processed value to the client instead of the three underlying fields (`last_edited_by_user_type`, `last_edited_by_user_key`, and `last_edited_by_support_user_email`), keeping the complexities of user identification encapsulated on the server.

I considered storing the single processed value in the DB instead of the three separate fields, but that would make it harder to change format down the road as well as to cross reference source-of-truth records.